module1.js

```diff
@@ -87,11 +98,12 @@ function question1(name1) {
   @returns the product of num1 and num2 
 */
 
-// function question5(num1, num2) {
-//   // TODO
+function question5(num1, num2) {
+     
+  // TODO
 
-//   return /* ??? */;
-// }
+  return Math.round((num1 * num2)*100)/100 
+}
 
```

Comment:
Good. Looks like you read the article I posted.

Reply:
okay thanks alot